src/command.coffee

```diff
@@ -185,7 +186,7 @@ compileScript = (file, input, base = null) ->
     else
       compiled = CoffeeScript.compile t.input, t.options
       t.output = compiled
-      if o.map
+      if o.map and not o['inline-map']
```

Comment:
@DylanPiercey @michaelficarra Browsers will not fail simply because there are both inline and external source maps. The behavior may be indeterminate, but that is not itself a good reason to prevent this use case. Besides, browsers are not the only consumer of compiled CoffeeScript.

I say give the developer the power to use both options simultaneously. And let's get this merged in.